dali/dfuXRefLib/dfuxreflib.cpp

```diff
@@ -2669,24 +2700,31 @@ IPropertyTree *  runXRef(unsigned nclusters,const char **clusters,IXRefProgressC
     if (nclusters==0)
         return NULL;
     CXRefManager xrefmanager;
-    const char *dirs[2];
-    unsigned numdirs;
 #ifdef _WIN32
     bool islinux = false;
 #else
     bool islinux = true;
 #endif
-    // assume all nodes same OS
-    Owned<IGroup> group = queryNamedGroupStore().lookup(clusters[0]);
 #ifdef _CONTAINERIZED
-    WARNLOG("CONTAINERIZED(runXRef calls queryOS())");
+    DBGLOG("CONTAINERIZED(runXRef)");
+    const char *dirs[nclusters]; // nclusters is the number of storage planes
+    unsigned numdirs = nclusters;
+
+    for (int i = 0; i < numdirs; i++)
+    {
+        Owned<IPropertyTree> storagePlane = getStoragePlane(clusters[i]);
+        dirs[i] = storagePlane->queryProp("@prefix");
```

Comment:
I am not sure it makes sense to be enumerating the directories into 'dirs' and passing them xrefmanager.process.

In BM, the dirs are not different cluster dirs, but the primary/replicate dirs, which are consistent across all clusters.

I think we should keep it simple in cloud version, and assert that we are only dealing with 1 plane at a time (at least initially), the dirs should contain a single directory of the plane prefix being processes

Reply:
Changed to only pass in a single directory through dirs.